lib/index.js

```diff
@@ -105,11 +105,14 @@ function observeStore(store, select, onChange) {
 
```

Comment:
line :95+ , let's add some debounce function, to prevent creating XXX calls eg. when writing dashboard's name